tests/test_app.py .....F                                                                                                                                                                                         [100%]

======================================================================================================= FAILURES =======================================================================================================
__________________________________________________________________________________________________ test_public_posts ___________________________________________________________________________________________________

    def test_public_posts():
>       posts_data = public_get_posts()

tests/test_app.py:70: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    @router.get('/posts')
    def public_get_posts():
>       return jsonplaceholder.public_get_posts()
E       AttributeError: module 'connectors.jsonplaceholder' has no attribute 'public_get_posts'

routes/jsonplaceholder.py:28: AttributeError
-------------------------------------------------------------------------- generated xml file: /home/visie/faculdade/TCC/tdd/test_results.xml --------------------------------------------------------------------------
=============================================================================================== short test summary info ================================================================================================
FAILED tests/test_app.py::test_public_posts - AttributeError: module 'connectors.jsonplaceholder' has no attribute 'public_get_posts'
============================================================================================= 1 failed, 5 passed in 4.17s ==============================================================================================
